index.js

```diff
@@ -14,21 +14,39 @@ var base64encoder = function (url, options, callback) {
     throw new Error('URL cannot be empty!');
   }
 
-  request({url: url, encoding: null}, function (err, res, body) {
-    if (err) { return callback(err); }
-
-    if (body && res.statusCode === 200) {
-      var image;
-
-      if (options && options.string === true) {
-        image = body.toString('base64');
-        return callback(null, image);
-      } else {
-        image = new Buffer(body, 'base64');
-        return callback(null, image);
-      }
+  var encoder = function (body, options) {
+    var image;
+
+    if (options && options.string === true) {
```

Comment:
only if you would like to return a string, to quote the readme: 
> if string is passed is with 'true', the image returned will be a base64 string. Otherwise, the base64 buffer is returned.